data_mapper_setup.rb

```diff
@@ -0,0 +1,6 @@
+require 'data_mapper'
+require './model/maker'
+require './model/peep'
+
+DataMapper.setup(:default, ENV['DATABASE_URL'] || "postgres://localhost/chitter_#{ENV['RACK_ENV']}")
```

Comment:
Metrics/LineLength: Line is too long. [100/90] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)